WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -264,30 +262,45 @@ class PostListFragment : Fragment() {
         scrollPosition?.let { recyclerView?.smoothScrollToPosition(it) }
     }
 
-    private fun updateEmptyViewForState(emptyViewState: PostListEmptyViewState) {
-        actionableEmptyView?.visibility = if (emptyViewState == HIDDEN_LIST) View.GONE else View.VISIBLE
-        val stringId = when (emptyViewState) {
-            HIDDEN_LIST -> return // nothing else to do!
-            EMPTY_LIST -> R.string.posts_empty_list
-            LOADING -> R.string.posts_fetching
-            REFRESH_ERROR -> if (NetworkUtils.isNetworkAvailable(nonNullActivity)) {
-                R.string.error_refresh_posts
-            } else R.string.no_network_message
-            PERMISSION_ERROR -> R.string.error_refresh_unauthorized_posts
-        }
-        val isNewPostActionVisible = emptyViewState == EMPTY_LIST
-        actionableEmptyView?.let {
-            it.image.setImageResource(R.drawable.img_illustration_posts_75dp)
-            it.image.visibility = if (isNewPostActionVisible) View.VISIBLE else View.GONE
-            it.title.setText(stringId)
-            it.button.setText(R.string.posts_empty_list_button)
-            it.button.visibility = if (isNewPostActionVisible) View.VISIBLE else View.GONE
-            it.button.setOnClickListener {
-                viewModel.newPost()
+    private fun updateEmptyViewForState(state: PostListEmptyUiState) {
+        if (state.listVisible) {
+            actionableEmptyView?.visibility = View.GONE
```

Comment:
I believe this needs to be exactly the opposite. See my comment in `PostListEmptyUiState.listVisible` for why that's the case.

Here is how my post list looks like in this state: https://cloudup.com/cp5-WEa70H1. Here is how it looks like after I make that change: https://cloudup.com/cdQa-SyCZ-u

Reply:
You are right:X. I'm totally confused now. How come I haven't noticed this during the tests/while I was taking the screenshots ... I might have broken it during a last minute change 😕

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9287/commits/9e1f8628c383e44886678f16378f9616f341825a

Reply:
Btw I've also noticed the empty views in https://cloudup.com/cdQa-SyCZ-u don't look right -> did you record the gif on this branch?

Reply:
Oh, maybe I made a mistake. I am actually fairly sure that I did, because I changed that code to record those gifs, but maybe I didn't pull the latest changes or something, I am not sure.